.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,22 +1,25 @@
 ---
 name: Bug report
-about: Create a report to help us improve
+about: Create a report to help us improve.
 title: ''
 labels: ''
```

Comment:
Add 'bug' here Will automatically add the label

Reply:
Unable to understand this. Please explain.